vocabularies/quality.ttl

```diff
@@ -0,0 +1,685 @@
+  skos:inScheme dq-iso-25012:Dimensions ;
+  skos:prefLabel "Confidentiality"@en ;
+  dqv:inCategory dq-iso-25012:inherentSystemDependentDataQuality ;
+  skos:broader dq-iso-25012:inherentSystemDependentDataQuality ;
+.
+dq-iso-25012:consistency
+  a dqv:Dimension ; a skos:Concept ;
+  dcterms:source <https://www.w3.org/TR/vocab-dqv/#DimentsionsMetricsHints> ;
+  rdfs:label "Consistency"@en ;
+  skos:definition "The degree to which data has attributes that are free from contradiction and are coherent with other data in a specific context of use. It can be either or both among data regarding one entity and across similar data for comparable entities."@en ;
+  skos:inScheme dq-iso-25012:Dimensions ;
+  skos:prefLabel "Consistency"@en ;
+  dqv:inCategory dq-iso-25012:inherentDataQuality ;
+  skos:broader dq-iso-25012:inherentDataQuality ;
+.
+dq-iso-25012:credibility
+  a dqv:Dimension ; a skos:Concept ;
+  dcterms:source <https://www.w3.org/TR/vocab-dqv/#DimentsionsMetricsHints> ;
+  rdfs:label "Credibility"@en ;
+  skos:definition "The degree to which data has attributes that are regarded as true and believable by users in a specific context of use. Credibility includes the concept of authenticity (the truthfulness of origins, attributions, commitments)."@en ;
+  skos:inScheme dq-iso-25012:Dimensions ;
+  skos:prefLabel "Credibility"@en ;
+  dqv:inCategory dq-iso-25012:inherentDataQuality ;
+  skos:broader dq-iso-25012:inherentDataQuality ;
+.
+dq-iso-25012:currentness
+  a dqv:Dimension ; a skos:Concept ;
+  dcterms:source <https://www.w3.org/TR/vocab-dqv/#DimentsionsMetricsHints> ;
+  rdfs:label "Currentness"@en ;
+  skos:definition "The degree to which data has attributes that are of the right age in a specific context of use."@en ;
+  skos:inScheme dq-iso-25012:Dimensions ;
+  skos:prefLabel "Currentness"@en ;
+  dqv:inCategory dq-iso-25012:inherentDataQuality ;
+  skos:broader dq-iso-25012:inherentDataQuality ;
+.
+dq-iso-25012:efficiency
+  a dqv:Dimension ; a skos:Concept ;
+  dcterms:source <https://www.w3.org/TR/vocab-dqv/#DimentsionsMetricsHints> ;
+  rdfs:label "Efficiency"@en ;
+  skos:definition "The degree to which data has attributes that can be processed and provide the expected levels of performance by using the appropriate amounts and types of resources in a specific context of use."@en ;
+  skos:inScheme dq-iso-25012:Dimensions ;
+  skos:prefLabel "Efficiency"@en ;
+  dqv:inCategory dq-iso-25012:inherentSystemDependentDataQuality ;
+  skos:broader dq-iso-25012:inherentSystemDependentDataQuality ;
+.
+dq-iso-25012:inherentDataQuality
+  a dqv:Category ; a skos:Concept ;
+  dcterms:source <https://www.w3.org/TR/vocab-dqv/#DimentsionsMetricsHints> ;
+  rdfs:label "Inherent data quality"@en ;
+  skos:prefLabel "Inherent data quality"@en ;
+  skos:inScheme dq-iso-25012:Dimensions ;
+.
+dq-iso-25012:inherentSystemDependentDataQuality

```

Comment:
No definition or documentation of meaning or examples.

Reply:
updated from https://iso25000.com/index.php/en/iso-25000-standards/iso-25012